packages/postgres-database/src/simcore_postgres_database/models/resource_tracker.py

```diff
@@ -3,13 +3,20 @@
     - Table where we store the resource usage of each container that
     we scrape via resource-usage-tracker service
 """
+import enum
 
 import sqlalchemy as sa
 from sqlalchemy.dialects.postgresql import JSONB
 
 from ._common import column_modified_datetime
 from .base import metadata
 
+
+class ContainerType(enum.Enum):
```

Comment:
- inherit from `(str, enum.Enum)` . all str enums in the db should do so.
- You might want to`StrAutoEnum` in https://github.com/ITISFoundation/osparc-simcore/blob/99db5b7afec0e60ef93ad06c6bd9fa45b0382f3b/packages/models-library/src/models_library/utils/enums.py.

Reply:
thanks!